### ISSUE TITLE
Explore Arrays class

### ISSUE THREAD

**Issue by sairohith2605** [AVNIETTech/Java-Core#2]
Develop code samples to implement various methods of the **Arrays** class.

**Comment by Datta973** [comment]
Yeah, i guess i am done with it.. But where would you like me to commit that ?
Shall i make a new folder saying "Arrays_Methods" 
I think we also need to create a branch

**Comment by sairohith2605** [comment]
Yes Datta, create a branch from master. Then feel free to upload your project folder, in case you're not using git tool, to that branch.
